sporkify/frontend/views.py

```diff
@@ -99,7 +99,12 @@ def inventory(request):
     if request.method == 'POST':
         entry = InventoryForm(request.POST)
         if entry.is_valid():
+            # Save the new item into the database
             entry.save()
+            # Remove the assigned code from open codes
+            code_to_remove = request.POST.get('product_code')
+            code_object = Open_Product_Code.objects.get(pk=code_to_remove)
```

Comment:
How does the `Open_Product_Code` you deleted get set on the `Inventory` item `product_code` field that's added through the `InventoryForm`?

Reply:
Ah, wait - I see. It gets provided in the render context and put into the field.